packages/dropbox/dropbox.0.1~772c4a6/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+maintainer: "Romain Beauxis <toots@rastageeks.org>"
+authors: "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+homepage: "https://github.com/Chris00/ocaml-docker"
+bug-reports: "https://github.com/Chris00/ocaml-dropbox/issues"
+license: "LGPL-3.0 with OCaml linking exception"
+dev-repo: "https://github.com/Chris00/ocaml-dropbox.git"
+build: [make]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "dropbox"]
+depends: [
+  "ocamlfind" "oasis" "lwt" "ssl" "cohttp" "uri" "atdgen"
```

Comment:
I suppose, and the same for the other deps. Looks like the requirement on OCaml >= 4.02.0 takes care of all of them..